internal/controller/controllers/agent_controller.go

```diff
@@ -299,7 +444,18 @@ func (r *AgentReconciler) updateStatus(ctx context.Context, log logrus.FieldLogg
 		}
 
 		if h.Progress != nil && h.Progress.CurrentStage != "" {
-			agent.Status.Progress.CurrentStage = h.Progress.CurrentStage
+			var isNoneHostRebooting bool
+
+			if isNoneHostRebooting, err = r.isDay2NonePlatformHostRebooting(ctx, agent, h); err != nil {
```

Comment:
We can move the variable assignment outside the if statement so that we can reuse it in L492

Reply:
OK